app/src/main/java/ch/epfl/qedit/viewmodel/QuizViewModel.java

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.qedit.viewmodel;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+import ch.epfl.qedit.backend.database.DatabaseFactory;
+import ch.epfl.qedit.model.Quiz;
+import ch.epfl.qedit.util.BundledData;
+import ch.epfl.qedit.util.Callback;
+import ch.epfl.qedit.util.Response;
+
+public class QuizViewModel extends ViewModel {
+    public enum Status {
+        NotLoaded,
+        Loading,
+        Loaded,
+        CouldntLoad
```

Comment:
"CouldNotLoad" would be better, no?

Reply:
I think that CouldntLoad stands for "Couldn't Load" but I prefer your version as well. I'm going to change it.